content/source/layouts/extend.erb

```diff
@@ -11,7 +11,7 @@
       </li>
       <li<%= sidebar_current("docs-extend-writing-custom-providers") %>>
         <!-- possibly make this "Guides" and collect some? -->
-        <a href="/docs/extend/writing-custom-providers.html">Writing Custom Providers</a>
+        <a href="https://learn.hashicorp.com/terraform/providers/setup-implement-read">Writing Custom Providers</a>
```

Comment:
Let's add a little "going to external site" arrow, just to reduce surprise: 

```suggestion
        <a href="https://learn.hashicorp.com/terraform/providers/setup-implement-read">Writing Custom Providers <span class="glyphicon glyphicon-new-window"></span></a>
```

That comes out like this bit, from the TFC docs sidebar: 

![image](https://user-images.githubusercontent.com/484309/88584651-d333f400-d006-11ea-87dc-f490962faa54.png)